frontend/src/pages/Account.tsx

```diff
@@ -0,0 +1,280 @@
+import {Trans, useTranslation} from "react-i18next";
+import {useEffect, useState} from "react";
+import {
+    AlertColor,
+    Box,
+    Container, Typography,
+} from "@mui/material";
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Unstable_Grid2';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import {DateField} from "@mui/x-date-pickers";
+import Chip from '@mui/material/Chip';
+import Paper from '@mui/material/Paper';
+import {styled} from '@mui/material/styles';
+import Alert from '@mui/material/Alert';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import CircularProgress from '@mui/material/CircularProgress';
+import Stack from '@mui/material/Stack';
+import {AdapterDayjs} from '@mui/x-date-pickers/AdapterDayjs';
+import {LocalizationProvider} from '@mui/x-date-pickers/LocalizationProvider';
+import dayjs from "dayjs";
+
+import {API_BASE_URL, signIn, signOut} from '../components/Api';
+import {Province, ChipData, chipColor, User} from "../components/Types";
+import httpClient from "../httpClient";
+import {GetProvincesWithNA} from "../components/Provinces";
+
+export const Account = () => {
+    const {t} = useTranslation();
+
+    const [province, setProvince] = useState<Province | null>(null);
+    const [chipData, setChipData] = useState<readonly ChipData[]>([]);
+
+    const [alert, setAlert] = useState(false);
+    const [alertType, setAlertType] = useState<AlertColor>("error");
+    const [alertContent, setAlertContent] = useState('');
```

Comment:
Since these states tend to be updated together in response to a single event/action, it might be worth using a reducer (see `useReducer`) which maintains all 3 in a single state.

Reply:
done